src/lib/Output.svelte

```diff
@@ -103,10 +105,17 @@
 					const rectB1 = spanB1.getBoundingClientRect();
 					const rectB2 = spanB2.getBoundingClientRect();
 
-					const x1 = (rectA1.left + rectA2.right) / 2 - rectOutput.left;
-					const y1 = rectA1.bottom - rectOutput.top + lineGap;
-					const x2 = (rectB1.left + rectB2.right) / 2 - rectOutput.left;
-					const y2 = rectB1.top - rectOutput.top - lineGap;
+					let x1 = (rectA1.left + rectA2.right) / 2 - rectOutput.left;
+					let y1 = Math.max(rectA1.bottom, rectA2.bottom) - rectOutput.top;
+					let x2 = (rectB1.left + rectB2.right) / 2 - rectOutput.left;
+					let y2 = Math.min(rectB1.top, rectB2.top) - rectOutput.top;
+
+					const correction = endpointCorrection / ((y2 - y1) / (x2 - x1));
+					x1 += correction;
+					y1 += lineGap;
+					x2 -= correction;
+					y2 -= lineGap;
```

Comment:
Do you prefer the above or
```suggestion
					endpointCorrection /= (y2 - y1) / (x2 - x1);
					x1 += endpointCorrection;
					y1 += lineGap;
					x2 -= endpointCorrection;
					y2 -= lineGap;
```
?